daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-deadletter.md

```diff
@@ -0,0 +1,79 @@
+---
+type: docs
+title: "Dead Letter Topics"
+linkTitle: "Dead Letter Topics"
+weight: 4000
+description: "Use dead letter topics to capture undeliverable messages"
+---
+
+## Introduction
+
+Dead letter topics are used to hold messages that cannot be delivered to any receiver. This eases the pressure on receivers by freeing them from dealing with poison messages, while allowing developers to write code that reads from the dead letter topic and either fix the message or abandon it completely.
+
+When a dead letter topic is set, any message that failed to be delivered to an app for a configured topic is put on the dead letter topic.
+
+Dapr enables dead letter topics for all of it's pub/sub components, even if the underlying system does not support this feature natively.
+
+## Configuring a dead letter topic through YAML
+
```

Comment:
The diagram below is an example of how dead letter topics work. First a message is sent from a publisher on an `orders` topic. Dapr receives the message on behalf of a subscriber application, however the `orders` topic message fails to be delivered to the `/checkout`endpoint on the application, even after retries. As a result of the failure to deliver, the message is forwarded to the 'poisonMessages' topic which delivers this to the `/failedMessages` endpoint to be processed, in this case on the same application. The '`failedMessages` processing code could drop the message or resend a new message.

<img src="/images/pubsub-dead-letter-topic.png" width=800 alt="Diagram showing the steps for dead letter topic messages in subscriptions">

Reply:
Added